scripts/deploy_hcloud.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+set -e
+git config --global push.default simple # we only want to push one branch — master
+git fetch --unshallow origin
+ssh-keyscan -H $IP >> ~/.ssh/known_hosts
+# add repo on vps as a repote
+git remote add production ssh://$USER@$IP/home/$USER/susi-server-git
+# push updates
+git push -f production HEAD:master
+# build and start susi server
+ssh $USER@$IP <<EOF
+  cd susi-server-git
+  bin/stop.sh
+  git submodule update --recursive --remote
+  git submodule update --init --recursive
+  ./gradlew build -x test
```

Comment:
You can execute the tests by doing
```
mkdir -p data/generic_skills/
touch data/generic_skills/media_discovery
```
before running the build. That should be enough for getting the tests done. If not, please show which tests fail.

Reply:
@norbusan The changes have been done. I've removed the skip test flag from Gradle build and now it is building successfully 🙂

Reply:
Thanks, I assume you have tested that it actually builds.